Error building on Android.

I installed latest version of this library and getting build issue on Android. It is working fine on iOS.

Version:

 "react-native": "0.64.3",
 "react-native-notifications": "^4.3.1",

Freshly installed yesterday and getting this issue.

Here is the error. Anyone have any idea?

`Execution failed for task ':react-native-notifications:compileReactNative60DebugJavaWithJavac'.
node_modules/react-native-notifications/lib/android/app/src/main/java/com/wix/reactnativenotifications/core/ReactAppLifecycleFacade.java:50: error: cannot find symbol
        return mReactContext.hasActiveReactInstance();
                            ^
  symbol:   method hasActiveReactInstance()
  location: variable mReactContext of type ReactContext
`

--- a/lib/android/app/src/main/java/com/wix/reactnativenotifications/core/ReactAppLifecycleFacade.java
+++ b/lib/android/app/src/main/java/com/wix/reactnativenotifications/core/ReactAppLifecycleFacade.java
@@ -47,7 +47,11 @@ public class ReactAppLifecycleFacade implements AppLifecycleFacade {
             return false;
         }
 
-        return mReactContext.hasActiveReactInstance();
+        try {
+            return mReactContext.hasActiveCatalystInstance();
+        } catch (Exception e) {
+            return mReactContext.hasActiveReactInstance();
+        }
     }
 
     @Override

